Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,198 @@
+        "query": "CommonSecurityLog\n| where DeviceVendor == \"Cyber-Ark\" \n| where DeviceProduct == \"Vault\" \n| where DeviceEventClassID in (295,428)| where FileName !contains \"PSMSessions\"| where FileName !contains \"PVWAConfig\"| where FileName !contains \"PasswordManagerShared\"| where FileName !contains \"VaultInternal\"| where FileName !contains \"PasswordManager\"| where FileName !contains \"PVWAPrivateUserPrefs\"| where FileName !contains \"ConjurSync\"| where FileName !contains \"SharedAuth_Internal\"| where FileName !contains \"PSM\"| where SourceUserName !contains \"PasswordManager\"| summarize count() by SourceUserName, TimeGenerated",
+        "size": 0,
+        "title": "Account objects accessed by user",
+        "noDataMessage": "It appears no accounts have been accessed",
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Account objects accessed"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\n| where DeviceVendor == \"Cyber-Ark\" \n| where DeviceProduct == \"Vault\" \n| where DeviceEventClassID in (302,359,360,361,412,411)\n| summarize audit=makeset(AdditionalExtensions) by ExternalID, DestinationUserName, SourceUserName",
+        "size": 0,
+        "title": "General audit information",
+        "noDataMessage": "There just isn't anything to show here",
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "gridSettings": {
+          "sortBy": [
+            {
+              "itemKey": "audit",
+              "sortOrder": 2
+            }
+          ]
+        },
+        "sortBy": [
+          {
+            "itemKey": "audit",
+            "sortOrder": 2
+          }
+        ]
+      },
+      "name": "Audit information"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where DeviceVendor == \"Cyber-Ark\" \n| where DeviceProduct == \"Vault\" \n| where DeviceEventClassID in (295,428)| where FileName contains \"ConjurSync\"| where SourceUserName contains \"Sync_components\"| summarize AggregatedValue = count() by bin(TimeGenerated, 1h)| sort by TimeGenerated desc| render timechart",
+        "size": 0,
+        "title": "Conjur Vault syncs",
+        "noDataMessage": "It doesn't look like you have Conjur",
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Conjur Vault syncs"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/e600a6ef-1740-4ee6-a6ba-3671a70986c3/resourcegroups/securitycenter/providers/microsoft.operationalinsights/workspaces/cyberark-devel"
```

Comment:
Remove the subscription ID from the workbook as this will be a publicly shared workbook.